refinery/data_set_manager/views.py

```diff
@@ -1059,11 +1060,20 @@ def post(self, request):
             return HttpResponseForbidden()
 
         logger.debug("Adding files to data set '%s'", data_set)
-        for file_store_item in data_set.get_file_store_items():
```

Comment:
Since this view now operates on single nodes and not on data sets it should be renamed accordingly along with the corresponding URL to reflect the change in functionality.

Reply:
Sure, sounds good